### ISSUE TITLE
Updates not compatible with lastest lovelace_gen

### ISSUE THREAD

**Issue by bratanon** [ludeeus/pyupdate#3]
**Version of the custom_component**
pyupdate 0.2.16

**Describe the bug**
When performing an update, `get_card_dir` will look for the string `!resource` in the main.yaml file if lovelace-gen is used. 

In the latest release of lovelave_gen.py that special directive is gone so users can't use that anymore.

**log**
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/homeassistant/components/websocket_api/decorators.py", line 17, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/local/lib/python3.6/site-packages/homeassistant/components/websocket_api/commands.py", line 148, in handle_call_service
    connection.context(msg))
  File "/usr/local/lib/python3.6/site-packages/homeassistant/core.py", line 1129, in async_call
    self._execute_service(handler, service_call))
  File "/usr/local/lib/python3.6/site-packages/homeassistant/core.py", line 1153, in _execute_service
    await self._hass.async_add_executor_job(handler.func, service_call)
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/custom_updater.py", line 94, in update_all_service
    card_controller.update_all()
  File "/config/custom_components/custom_updater.py", line 149, in update_all
    self.custom_url)
  File "/config/deps/lib/python3.6/site-packages/pyupdate/ha_custom/custom_cards.py", line 87, in update_all
    upgrade_single(base_dir, name, custom_repos)
  File "/config/deps/lib/python3.6/site-packages/pyupdate/ha_custom/custom_cards.py", line 97, in upgrade_single
    local_file = get_card_dir(base_dir, name) + name + '.js'
  File "/config/deps/lib/python3.6/site-packages/pyupdate/ha_custom/custom_cards.py", line 135, in get_card_dir
    card = line.split('!resource ')[1].split(name + '.js')
IndexError: list index out of range
```

**Comment by ludeeus** [comment]
The new method is `!file`, or am I missing something?

**Comment by bratanon** [comment]
`!file` does not seem to be the same. 

`!resource` would copy things from /lovelace to /www, thats not how `!file` is working

**Comment by ludeeus** [comment]
Do you mind sharing your resource section that `lovelace_gen ` uses?

**Comment by bratanon** [comment]
Having all LL-cards in www/lovelace

```
resources:
- url: /local/lovelace/card-tools.js?v=297fcc
  type: js
- url: /local/lovelace/tracker-card.js?v=0.2.0
  type: js
- url: /local/lovelace/layout-card.js?v=a313b7
  type: js
- url: /local/lovelace/slider-entity-row.js?v=ba8f50
  type: js
- url: /local/lovelace/mini-graph-card-bundle.js?v=0.1.0
  type: module
- url: /local/lovelace/monster-card.js?v=0.2.3
  type: js
- url: /local/lovelace/attribute-entity-row.js?v=0
  type: module
- url: /local/lovelace/secondaryinfo-entity-row.js?v=70b1cc
  type: module
```

**Comment by bratanon** [comment]
I have switch over to use the normal _ui-lovelace.yaml_ now since _!include_ and _!secret_ works there again. So this will not be a problem for me anymore. The bug is still there tho, however, the lovelace_gen project is some what deprecated now, Thomas himself stated this in an issue in lovelace_gen. 

> !include (with more variants) and !secret are already supported natively in ui-lovelace.yaml, and the templating is of limited use to most people. Since the pending removal of !Include support was the reason lovelace-gen even came to be, it's actually pretty much deprecated now anyway...
The remaining reason to use lovelace-gen is the templating, which - while convenient for advanced users - have limited use for most people and goes against the current path of lovelace development.

Source https://github.com/thomasloven/homeassistant-lovelace-gen/issues/10

**Comment by ludeeus** [comment]
Thanks @bratanon 
Corrected in 0.2.18
